package.json

```diff
@@ -10,8 +10,8 @@
     "build:assets": "node ./buildAssets.js",
     "commit": "superset-commit",
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s -r 0",
-    "type": "NODE_ENV=production beemo typescript --workspaces=\"@superset-ui/((preset|plugin)-!(chart-xy))\" --noEmit",
-    "type:dts": "NODE_ENV=production beemo typescript --workspaces=\"@superset-ui/((preset|plugin)-!(chart-xy))\" --emitDeclarationOnly",
+    "type": "NODE_ENV=production beemo typescript --workspaces=\"@superset-ui/((preset|plugin))\" --noEmit",
```

Comment:
nit: before it was `@superset-ui/((preset|plugin)-*)\`  if I remember correctly.